roles/collectd_config/.travis.yml

```diff
@@ -1,29 +0,0 @@
----
```

Comment:
The top level .travis.yml needs to either be removed, or updated to install ansible-galaxy, and make sure both roles are imported.

I'm not sure if it makes sense to do testing in this repo, or to what extent it makes sense, since we can configure travis tol run on the other repos anyway.

Reply:
That .travis was in the role, which is still in the role in the other repository. Just needs to be configured to run.